### ISSUE TITLE
Casing broken since the recent dashing changes

### ISSUE THREAD

**Issue by DerDakon** [kothic/kothic-js#60]
This is how OpenRailwayMap looks with the original rendering code:
![casing-old](https://cloud.githubusercontent.com/assets/10909049/13373971/0b68a852-dd77-11e5-9348-8ca422a8f724.png)

This is how it looks now:
![casing-new](https://cloud.githubusercontent.com/assets/10909049/13373970/0b60da14-dd77-11e5-936b-7423c876d64d.png)

The casing is now either always on or always off as it looks to me. The change is the new dashing code from @braandl.

**Comment by braandl** [comment]
Hm, I use the new code in my Application, and I can not find any of that behavior so far. When does this happen?
![no-dashing-issues](https://cloud.githubusercontent.com/assets/3504490/13389163/f877c12e-dec5-11e5-9cff-f03f8a2efa8b.JPG)

**Comment by DerDakon** [comment]
It's [this code](https://github.com/rurseekatze/OpenRailwayMap/blob/master/styles/standard.mapcss#L98).

**Comment by braandl** [comment]
Hm, when I define casing and dashing in the same selector, it works as intended, i think?
Maybe you should define the dashes in "way|z9-[railway=razed].tracks", too?

I am not sure what the correct behavour is...

![image](https://cloud.githubusercontent.com/assets/3504490/13424510/a75f4140-dfa1-11e5-96b4-17945cc9103a.png)

**Comment by DerDakon** [comment]
Your case looks correct.

**Comment by DerDakon** [comment]
Which version of Leaflet do you use? Which Browser?

**Comment by braandl** [comment]
Leaflet 0.7.7 and I did test it with Google Chrome, Firefox on Windows, Ubuntu , Opera on Windows and Mobile and Safari MacOsX 10.11 aswell as on iPhones with iOS 9.3 and on an iPad Air having no issues that I am aware of.

**Comment by braandl** [comment]
All the Browsers were always updated to the latest stable version.

**Comment by DerDakon** [comment]
We were still using Leaflet 0.6.2 back then, but meanwhile I upgraded. I'll test again.

**Comment by DerDakon** [comment]
![abandoned](https://cloud.githubusercontent.com/assets/10909049/15091450/ed1bdc02-1449-11e6-9089-8015cbbe7106.png)

South of the bridge this is just one way, which in the middle stops showing any casing. I changed the base color to green just to make sure there are no other styles interfere with that.

Have you tried your code when the line crosses tile borders?

**Comment by braandl** [comment]
Yes, it works on my Installation that provides tiles for entire regions as gjson

**Comment by braandl** [comment]
![multiple-tiles-casing](https://cloud.githubusercontent.com/assets/3504490/15091526/a4e7d132-144b-11e6-96b8-e1f1e0f4e786.JPG)

**Comment by Komzpa** [comment]
Closing this old rendering/debugging issue as stale for the legacy Kothic JS runtime. There is no current reproduction path or maintainer-owned fix in this repository; a revived fix should come as a fresh source-level PR with a working demo or focused regression test.

**Comment by Komzpa** [comment]
Reopening: I closed this too aggressively during cleanup triage. This should stay open until there is an explicit maintainer decision on whether to fix, supersede, migrate, or intentionally close it.
